repository-data/webfiles/src/main/resources/site/freemarker/hee/layout/base-layout.ftl

```diff
@@ -36,6 +36,8 @@
 
     <@hst.include ref="breadcrumb"/>
 
+    <@hst.include ref="hero"/>
```

Comment:
@thojava - If we move the following content layout wrapper back to document's main freemarker templates, I guess we might be able to move `hero` under `homepage-main.ftl` (after moving `hero` container under `/hst:hee/hst:configurations/lks/hst:pages/homepage/main`) so that it's part of home-page main template.
```
  <div class="nhsuk-width-container">
      <main id="maincontent" role="main" class="nhsuk-main-wrapper">
      </main>
  </div>
```

Also, this might help us to cache `hero` container under home main (rather than under base-layout). Does this makes sense ? Sorry, I yet to get my head around caching strategies of brXM.

Looking at http://localhost:3000/lks/home on the `hee-prototypes`, looks like content layout wrapper will have additional classes in order to accommodate alternating background colours for home sections for instance (though not sure this is indicated in figma):
```
  <div class="nhsuk-width-container nhsuk-width-container--full">
    <main class="nhsuk-main-wrapper nhsuk-main-wrapper--no-padding nhsuk-homepage" id="maincontent" role="main">
    </main>
  </div>
```

Let me know your thoughts.

Reply:
@smanivasag I think that is the only way to achieve the thing. We did that way last time let rollback ;)

Reply:
@thojava - Thanks for the rollback 👍